operations/power_management/Power_Off_Compute_Cabinets.md

```diff
@@ -67,11 +210,11 @@ liquid-cooled cabinet chassis, compute modules, and router modules, then powers
 
 1. (`ncn-m#`) Shut down cabinet power.
 
-    **Important:** The default timeout for the call to PCS is 120 seconds. If the `sat bootsys shutdown` command fails
-    to power off some cabinets and indicate that requests to PCS have timed out, the `sat` command may be run with an increased `--pcs-timeout` value.
+    **Important:** The default timeout for the call to CAPMC is 120 seconds. If the `sat bootsys shutdown` command fails
```

Comment:
In CSM 1.6 we've replaced all references to CAPMC with PCS.  The 'sat' command  has also replaced --capmc-timeout with --pcs-timeout in 1.6

In your 1.5 PR, keeping CAPMC and --capmc-timeout in place is correct.  Just need to be replaced in this 1.6 PR.

```suggestion
    **Important:** The default timeout for the call to PCS is 120 seconds. If the `sat bootsys shutdown` command fails
```

Reply:
Fixed only on 1.6